logging.js

```diff
@@ -83,15 +83,13 @@ export class ServerLogger {
 }
 
 export class LogBuilder {
+
 	constructor(appId) {
 		this._log = { appId: String(appId) };
 	}
 
-	withLocation() {
-		if (window.location && window.location.href) {
-			this._log.location = String(window.location.href);
-		}
-		return this;
+	build() {
+		return Object.freeze(this._log);
```

Comment:
These are all just alphabetizing method names, which is required as part of the new linting reules.